electrum/gui/qt/installwizard.py

```diff
@@ -297,7 +300,10 @@ def on_filename(filename):
                 self.pw_e.hide()
 
         button.clicked.connect(on_choose)
-        button2.clicked.connect(lambda: self.loop.exit(2))
+
+        button2.clicked.connect(partial(self.name_e.setText,get_new_wallet_name(wallet_folder)))
```

Comment:
Please follow PEP8 formatting guide. There are too many new lines and it's need for space around `,` in function call.

Reply:
done